src/nspawn/nspawn.c

```diff
@@ -2740,9 +2766,12 @@ static int outer_child(
         if (fd < 0)
                 return fd;
 
+        if (arg_private_network || arg_network_namespace_path)
+                use_netns = true;
```

Comment:
rewrite this as this maybe:

```c
use_netns = arg_private_network || arg_network_namespace_path;
```

That said, taking the comments further down into account this should probably be:

```
use_netns = !arg_network_namespace_path && arg_private_network;
```

Reply:
hmm, maybe rename the variable too? use_netns → create_netns?